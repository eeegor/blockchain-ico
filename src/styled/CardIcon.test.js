import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { CardIcon } from '.';

describe('CardIcon', () => {
	test('renders card icon', () => {
		const tree = renderer.create(<CardIcon>Icon</CardIcon>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
