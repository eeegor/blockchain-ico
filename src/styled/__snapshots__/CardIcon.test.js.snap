// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardIcon renders card icon 1`] = `
.c0 {
  max-width: 100%;
  height: 40px;
  width: 40px;
  background: #3866FF15;
  border-radius: 20px;
  overflow: hidden;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 12px;
  color: #fff;
}

<div
  className="c0"
>
  Icon
</div>
`;
